cmd/in_memory_channel_controller/main.go

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Is this purposely different than the existing `cmd/in_memory/controller/main.go`?

Reply:
yes, because it uses controller runtime.